pincer/objects/events/channel.py

```diff
@@ -28,3 +29,9 @@ class ChannelPinsUpdateEvent(APIObject):
 
     guild_id: APINullable[Snowflake] = MISSING
     last_pin_timestamp: APINullable[Timestamp] = MISSING
+
+
+    def __post_init__(self):
+        self.last_pin_timestamp = convert(
+            self.last_pin_timestamp, Timestamp.parse, Timestamp
+    )
```

Comment:
Post init's arn't required on `APIObject`'s as these should get casted automatically.